content/docs/introduction/faq.md

```diff
@@ -110,16 +123,7 @@ various components attempt to handle failing changes gracefully.
 
 Yes, with the [Alertmanager](https://github.com/prometheus/alertmanager).
 
-Currently, the following external systems are supported:
-
-* Email
-* Generic Webhooks
-* [OpsGenie](https://www.opsgenie.com/)
-* [PagerDuty](http://www.pagerduty.com/)
-* [Pushover](https://pushover.net/)
-* [Slack](https://slack.com/)
-* [VictorOps](https://victorops.com/)
-* [WeChat](https://www.wechat.com)
+We support sending alerts through email and a webhook system. For currently supported systems, please see our [integrations page](https://prometheus.io/docs/operating/integrations/#alertmanager-webhook-receiver).
```

Comment:
There is more natively supported which are not listed in the linked page

Reply:
```suggestion
We support sending alerts through email, native integrations, and a webhook system.

Native integrations:
* [OpsGenie](https://www.opsgenie.com/)
* [PagerDuty](http://www.pagerduty.com/)
* [Pushover](https://pushover.net/)
* [Slack](https://slack.com/)
* [VictorOps](https://victorops.com/)
* [WeChat](https://www.wechat.com)
 
For systems supported through webhook, please see our [integrations page](https://prometheus.io/docs/operating/integrations/#alertmanager-webhook-receiver).
```

Reply:
While I commend the effort of listing all Alertmanager notification integrations here, I think that's bound to get out of date... probably we should just link to https://prometheus.io/docs/alerting/latest/configuration/?

Reply:
it is already out of date

Reply:
We could say "for direct integrations go here, for webhook go there" and done.

Reply:
```suggestion
We support sending alerts through email and a webhook system. For currently supported systems, please see our [integrations page](https://prometheus.io/docs/operating/integrations/#alertmanager-webhook-receiver).


We support sending alerts through [email, various native integrations](https://prometheus.io/docs/alerting/latest/configuration/), and a [webhook system anyone can add integrations to](https://prometheus.io/docs/operating/integrations/#alertmanager-webhook-receiver).
```

Reply:
SGTM, but the first paragraph (line 126 in your snippet above) needs to be removed?

Reply:
```suggestion
We support sending alerts through [email, various native integrations](https://prometheus.io/docs/alerting/latest/configuration/), and a [webhook system anyone can add integrations to](https://prometheus.io/docs/operating/integrations/#alertmanager-webhook-receiver).
```